specification/attestation.adoc

```diff
@@ -1,309 +1,874 @@
+following claims:
+
+[caption="Figure {counter:image}: ", reftext="Figure {image}"]
+[title= "Platform Claims Map"]
+[source,cddl]
+----
+platform-token = #6.61(platform-token-map)
+
+platform-token-map = {
+   riscv-ap-tee-eat-profile ; EAT Profile
+   platform-challenge       ; Hash of the TSM public key
+   platform-public-key      ; Platform public key
+   platform-manufacturer-id ; Platform manufacturer identifier
+   platform-state           ; Platform operational state
+   platform-sw-components   ; Platform SW components
+}
+----
+
+All above described Claims must be present in the Platform Token.
+
+====== EAT Profile Claim
+
+The platform EAT profile claim describes the EAT profile that the AP-TEE
+platform implements. The profile should include a description of all three
+tokens (platform, TSM and TVM) as they are bound together.
+
+[caption="Figure {counter:image}: ", reftext="Figure {image}"]
+[title= "EAT Profile Claim"]
+[source,cddl]
+----
+riscv-ap-tee-eat-profile-label = 265 ; EAT profile
+riscv-ap-tee-eat-profile-doc = "https://riscv.org/TBD"
+
+riscv-ap-tee-eat-profile = (
+   riscv-ap-tee-eat-profile-label => riscv-ap-tee-eat-profile-doc
+)
+----
+
+====== Platform Challenge Claim
+
+The platform challenge claim is the hash value of the TSM public key, that is
+used to sign the TVM token.
+
+[caption="Figure {counter:image}: ", reftext="Figure {image}"]
+[title= "Platform Challenge Claim"]
+[source,cddl]
+----
+platform-challenge-label = 10 ; EAT nonce
+riscv-ap-tee-hash-type =
+                       bytes .size 32 / ; SHA256
+                       bytes .size 48 / ; SHA384
+                       bytes .size 64   ; SHA512
+
+platform-challenge = (
+   platform-challenge-label => riscv-ap-tee-hash-type
+)
+----
+
+====== Platform Public Key Claim
+
+The platform public key claim identifies the platform public key that signs the
+TSM token. The value of the claim is a COSE_Key <<COSE>> structure encoded as a
+CBOR bytes string.
+
+[caption="Figure {counter:image}: ", reftext="Figure {image}"]
+[title= "Platform Public Key Claim"]
+[source,cddl]
+----
+platform-public-key-label = TBD
+riscv-ap-tee-public-key-type = bytes .cbor COSE_Key
+
+platform-public-key = (
+   platform-public-key-label => riscv-ap-tee-public-key-type
+)
+----
+
+====== Platform Manufacturer Identifier Claim
+
+The platform manufacturer identifier claim uniquely identifies the manufacturer
+for the AP-TEE implementation providing the Attestation Evidence.
+Verification services can use the value of this claim to obtain the manufacturer
+provided Endorsements for the platform implementation.
+
+[caption="Figure {counter:image}: ", reftext="Figure {image}"]
+[title= "Platform Manufacturer Identifier Claim"]
+[source,cddl]
+----
+platform-manufacturer-id-label = TBD
+platform-manufacturer-id-type = .bytes 64
+
+platform-manufacturer-id = {
+   platform-manufacturer-id-label => platform-manufacturer-id-type
+}
```

Comment:
here instead you probably want a group rather than a map

```suggestion
platform-manufacturer-id = (
   platform-manufacturer-id-label => platform-manufacturer-id-type
)
```

Reply:
And that is a group now, thanks.